README.md

```diff
@@ -29,6 +30,20 @@ applications. Applications request a rate limit decision based on a domain and a
 reads the configuration from disk via [runtime](https://github.com/lyft/goruntime), composes a cache key, and talks to the redis cache. A
 decision is then returned to the caller.
 
+# Deprecation of Legacy Ratelimit Proto
+
+Envoy's data-plane-api defines a ratelimit service proto [rls.proto](https://github.com/envoyproxy/data-plane-api/blob/master/envoy/service/ratelimit/v2/rls.proto).
+Logically the data-plane-api [rls](https://github.com/envoyproxy/data-plane-api/blob/master/envoy/service/ratelimit/v2/rls.proto)
+is equivalent to the [ratelimit.proto](https://github.com/lyft/ratelimit/blob/0ded92a2af8261d43096eba4132e45b99a3b8b14/proto/ratelimit/ratelimit.proto)
+defined in this repo. However, due
+to the namespace differences and how gRPC routing works it is not possible to transparently route the
+legacy ratelimit (ones based in the [ratelimit.proto](https://github.com/lyft/ratelimit/blob/0ded92a2af8261d43096eba4132e45b99a3b8b14/proto/ratelimit/ratelimit.proto)
+defined in this repo) requests to the data-plane-api
+definitions. Therefore, the ratelimit service will upgrade the requests, process them internally as it would
+process a data-plane-api ratelimit request, and then downgrade the response to send back to the client. This means that,
+for a slight performance hit for clients using the legacy proto, ratelimit is backwards compatible with the legacy proto.
```

Comment:
Are we supporting it long-term, or just for this release? IIRC, the discussions around this were to yank out the old proto support with next release, correct? If so, I'd recommend adding a note to that effect.